### ISSUE TITLE
Bug: no-invalid-named-grid-areas does not recognize tabs as whitespace between cell tokens

### ISSUE THREAD

**Issue by KumJungMin** [eslint/css#537]
### Environment

ESLint version: 10.8.0
@eslint/css version: 1.4.0
Node version: 22.18.0
npm version: 10.9.3
Operating System:

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```js
import { defineConfig } from "eslint/config";
import css from "@eslint/css";

export default defineConfig([
    {
        files: ["**/*.css"],
        language: "css/css",
        plugins: { css },
        rules: {
            "css/no-invalid-named-grid-areas": "error",
        },
    },
]);
```

</details>

```css
.grid {
    grid-template-areas:
        "a	b"
        "a b";
}
```
The first row contains a tab character between a and b, while the second row contains a space.

### What did you expect to happen?

No lint error.

According to the CSS Grid specification, a sequence of whitespace separates cell tokens in a grid-template-areas string. Since a tab is a CSS whitespace character, both rows should contain two cell tokens:

```md
"a<TAB>b" → ["a", "b"]
"a b"     → ["a", "b"]
```
Therefore, both rows define the same number of grid cells.

### What actually happened?

no-invalid-named-grid-areas reports that the second row has a different number of cell tokens:

```md
4:5  error  Grid area strings must have the same number of cell tokens  css/no-invalid-named-grid-areas
```

The rule currently splits each grid row using a literal space:

```js
const row = trimmedValue.split(" ").filter(Boolean);
```

As a result, a tab is not treated as a separator:
```md
"a<TAB>b" → ["a<TAB>b"]
"a b"     → ["a", "b"]
```

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/vitejs-vite-zfq5zwpy?file=test.css

### Participation

- [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [ ] I did not use AI to generate this issue report.
- [x] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

> Disclosure: I'm a participant in the [OSSCA open source contribution program](https://github.com/eslint-ossca)

According to the CSS Grid specification, strings in `grid-template-areas` are tokenized using longest-match semantics, and a sequence of whitespace does not produce a token:

<img width="1315" height="401" alt="Image" src="https://github.com/user-attachments/assets/20cef9bc-cd2d-4fe3-971c-da8f019c5f8a" />

https://www.w3.org/TR/css-grid/#grid-template-areas-property

<br/>

The CSS specification includes U+0009 CHARACTER TABULATION (tab) as whitespace:

<img width="1163" height="132" alt="Image" src="https://github.com/user-attachments/assets/b9d1bbd6-7d42-46bc-a76b-18b87cf9524f" />

https://www.w3.org/TR/css-syntax-3/#whitespace

<br/>

Therefore, both `"a b"` and `"a<TAB>b"` should be parsed as two cell tokens, `a` and `b`.

The rule currently splits each row using a literal U+0020 SPACE:

```js
const row = trimmedValue.split(" ").filter(Boolean);

**Comment by xevemuy** [comment]
Please stop spamming issues.

**Comment by Pixel998** [comment]
Thanks for the report. I agree this is is a bug. PR is welcome.
